src/index.js

```diff
@@ -62,7 +63,16 @@ export const isPluginRequired = (supportedEnvironments, plugin) => {
 };
 
 const getTargets = targetOpts => {
-  return targetOpts || {};
+  const mergedOpts = targetOpts || {};
+  const browserOpts = targetOpts['browsers'];
```

Comment:
Do we want to do any validation here? (like in loose/modules we verify if boolean etc)